code/app/src/main/res/layout/recipe_add_ingredients_fragment_layout.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="32dp">
+
```

Comment:
Box background color inconsistent

Reply:
yeah rn a placeholder, ill try and imitate the user interface later

Reply:
fixed^